politeiawww/politeiawww.go

```diff
@@ -971,7 +971,14 @@ func (p *politeiawww) setupPiDcrdataWSSubs() error {
 			case *exptypes.WebsocketBlock:
 				log.Debugf("wsDcrdata message WebsocketBlock(height=%v)",
 					m.Block.Height)
-				p.updateBestBlock(uint64(m.Block.Height))
+				bb := uint64(m.Block.Height)
+				p.updateBestBlock(bb)
+				// Keep VoteResults table updated with received best block
+				_, err = p.decredLoadVoteResults(bb)
+				if err != nil {
+					log.Errorf("decredLoadVoteResults: %v", err)
+					return
```

Comment:
What's your reason for returning here?

Reply:
Hmmm yeah, I guess it does not make sense to return from the ws conn when load vote results fails. Removing this.